assets/tagRenderings/questions.json

```diff
@@ -629,7 +629,23 @@
     "freeform": {
       "key": "opening_hours",
       "type": "opening_hours"
-    }
+    },
+    "mappings": [
```

Comment:
I don't think adding 24/7 is a good idea, this will also show up at many amenities for which 24/7 doesn't make sense (such as for cafés, restaurants, shops, ...).

It would be better to have a new shared-tagRendering, e.g. "opening_hours_24_7" that can be reused. Note that you can reuse tagrenderings here as well!